Factorio.asl

```diff
@@ -146,3 +430,8 @@ isLoading {
 shutdown {
 	vars.splitsDone = new HashSet<string>();
 }
+reset {
+	if(memory.ReadValue<byte>((System.IntPtr)(current.gamePointer)) == 0){
```

Comment:
I was wondering about whether or not we should include this. Perhaps we can make the default "reset" value in the livesplitter false? (If that is even possible?)

At least when I run the coop Any% we sometimes have a disconnect, and if that resets a timer that would be very unfortunate... ;(